types/cookie_js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for cookie_js v1.2.2
+// Type definitions for cookie_js v1.2.3
```

Comment:
Can you enable the disabled lint rules in `tslint.json`? Typically our lint rules should pick up on an inappropriate version number.

Reply:
To elaborate a bit more - we only accept a version number like `1.2`, not `v1.2.3`

Reply:
@DanielRosenwasser 

I should not have changed the version. sorry I didn't understand correctly, I've removed the commit. 

>Can you enable the disabled lint rules in tslint.json?

regarding this, how should I change?

Reply:
ping @DanielRosenwasser